subsystems/chassis/SolenoidShifters.java

```diff
@@ -1,15 +1,15 @@
 package org.usfirst.frc4904.standard.subsystems.chassis;
 
-
 import org.usfirst.frc4904.standard.commands.chassis.ChassisShift;
 import edu.wpi.first.wpilibj.DoubleSolenoid;
-import edu.wpi.first.wpilibj.command.Subsystem;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 /**
  * A subsystem for managing a solenoid for a shifting drivetrain.
  *
  */
-public class SolenoidShifters extends Subsystem {
+public class SolenoidShifters extends SubsystemBase {
```

Comment:
should be `implements Subsystem` with `Subsystem` from `edu.wpi.first.wpilibj2.command.Subsystem`. This will also require changes to imports and the rest of the class